qiskit_dynamics/models/frame.py

```diff
@@ -672,6 +696,25 @@ def operators_into_frame_basis_with_cutoff(
 
         # create difference matrix for diagonal elements
         dim = len(ops_in_frame_basis[0])
+        cutoff_array = self.calculate_cutoff_filter(carrier_freqs, cutoff_freq, dim)
+
+        return (
+            cutoff_array * ops_in_frame_basis,
+            cutoff_array.transpose([0, 2, 1]) * ops_in_frame_basis,
+        )
+
+    def calculate_cutoff_filter(
```

Comment:
I wonder if it makes sense to just have this be a utility function in the file that contains the rotating wave approximation, since this is very specific to that.

Reply:
I think this function and operators_into_frame_with_cutoff are now obsolesced by the new Model -> Model RWA functions. I had originally used this because I wanted to be able to access _just_ this functionality from outside the Frame object in pervious versions. As of now, I'm not particularly attached of having this be a separate function.

Reply:
I think this function and operators_into_frame_with_cutoff are now obsolesced by the new Model -> Model RWA functions. I had originally used this because I wanted to be able to access _just_ this functionality from outside the Frame object in pervious versions. As of now, I'm not particularly attached of having this be a separate function.